org.eclipse.jdt.launching/launching/org/eclipse/jdt/launching/AbstractVMInstall.java

```diff
@@ -43,6 +44,7 @@
 import org.eclipse.jdt.internal.launching.Standard11xVM;
 import org.eclipse.jdt.internal.launching.StandardVMType;
 import org.eclipse.osgi.util.NLS;
+import org.graalvm.nativeimage.Platform;
```

Comment:
import gone wrong

Reply:
Ups. Probably went wrong during rebase. Fixed it. Thanks.